src/Workspaces/Remote/ServiceHub/Services/ServiceHubServiceBase.cs

```diff
@@ -25,9 +30,15 @@ protected ServiceHubServiceBase(Stream stream, IServiceProvider serviceProvider)
             Logger.TraceInformation($"{DebugInstanceString} Service instance created");
 
             Stream = stream;
```

Comment:
we can hide stream I think now, since JsonRpc should own it. if someone else uses it it will mess up JsonRpc.